integration-cli/requirements_unix.go

```diff
@@ -36,6 +36,19 @@ var (
 		},
 		"Test requires an environment that supports cgroup cfs quota.",
 	}
+	cpuShare = testRequirement{
```

Comment:
Might be helpful to use the sysinfo package for these requirements since this seems like it's all duplicated.